microsoft-edge/webview2/release-notes/index.md

```diff
@@ -16,18 +16,91 @@ These Release Notes provide information about new features and bug fixes that ar
 the template to copy for incoming sections is at end of this file
 
 this webpage covers the most recent ~5 months; periodically move oldest h2 sections from bottom of present file to archive.md.  eg covers: 
+Feb 2024
 Jan 2024
 [Dec 2024]
 Nov 2024
 Oct 2024
 Sep 2024
-Aug 2024
 
 moved to archive:
-Jul[/Aug] 2024
+Aug 2024
 -->
 
 
+<!-- Feb 2025 Release SDK -->
+<!-- ====================================================================== -->
+## 1.0.3065.39
+
+Release Date: February 10, 2025
+
+[NuGet package for WebView2 SDK 1.0.3065.39](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.3065.39)
+
+For full API compatibility, this Release version of the WebView2 SDK requires WebView2 Runtime version 133.0.3065.39 or higher.
+
+
+<!-- ------------------------------ -->
+#### Promotions
+
+No additional APIs have been promoted to Stable and added in this Release SDK.
+
+
+<!-- ------------------------------ -->
+#### Bug fixes
+
+
+<!-- ---------- -->
+###### Runtime-only
+
+* Added the missing **Close** button in the **Download** flyout.
+* Fixed a race condition that occurred when the Web Request Response event never occurs.
+
+<!-- end of Feb 2025 Release SDK -->
+
+
+<!-- Feb 2025 Prerelease SDK -->
+<!-- ====================================================================== -->
+## 1.0.3116-prerelease
+
+Release Date: February 10, 2025
+
+[NuGet package for WebView2 SDK 1.0.3116-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.3116-prerelease)
+
+For full API compatibility, this Prerelease version of the WebView2 SDK requires the WebView2 Runtime that ships with Microsoft Edge version 134.0.3116.0 or higher.
+
+
+<!-- ------------------------------ -->
+#### Experimental APIs
+
+No Experimental APIs have been added in this Prerelease SDK.
+
+
+<!-- ------------------------------ -->
+#### Promotions
+
+No APIs have been promoted from Experimental to Stable in this Prerelease SDK.
+
+
+
+<!-- ------------------------------ -->
+#### Bug fixes
+
+
+<!-- ---------- -->
+###### Runtime-only
+
+* Added the missing **Close** button in the **Download** flyout.
+* Fixed a race condition that occurred when the Web Request Response event never occurs.
+
+
+<!-- ---------- -->
+###### SDK-only
+
+* Fixed documentation of the `FindNext` API that incorrectly referenced `FindPrevious`.  ([Issue #5059](https://github.com/MicrosoftEdge/WebView2Feedback/issues/5059))
```

Comment:
I don't know if we usually call out reference content fix in the release notes. I don't feel strong if we decide to keep this. Reference content fix on lmc can be independent from SDK release.

Reply:
https://github.com/MicrosoftEdge/WebView2Feedback/issues/5059

Reply:
keeping.  might link to API Ref for 1-3 platforms